galeracluster/source/selinux.rst

```diff
@@ -0,0 +1,128 @@
+=======================
+SELinux Configuration
+=======================
+.. _`selinux`:
+
+Security-Enhanced Linux, or SELinux, is a kernel module for improving security on Linux operating system.  It integrates support for access control security policies, including mandatory access control (MAC), that limit user applications and system daemons access to files and network resources.  Some Linux distributions, such as Fedora, ship with SELinux enabled by default.
+
+In the context of Galera Cluster, systems with SELinux may block the database server, keeping it from starting or preventing the node from establishing connectivity with the cluster.  To avoid this, you need to configure SELinux policies to allow the node to operate.
+
+
+-----------------------------
+Generating an SELinux Policy
+-----------------------------
+.. _`gen-selinux-policy`:
+
+In order to create an SELinux policy for Galera Cluster, you need to first open ports and set SELinux into permissive mode.  Then, generate after generating various replication event, state transfers and notifications, create a policy from this activity, then restore SELinux.
```

Comment:
I do not understand "generate after generating"

replication events [plural]

restore SELinux must be "enable SELinux" or "set it to "enforcing"" or something along those lines.